src/main.cpp

```diff
@@ -213,29 +176,13 @@ class Demo {
 				std::cout << cappedFPS << " | " << uncappedFPS << "    \r" << std::flush;
 			}
 			
-			// Display audio debug information once a second
+			// Keep track of every second
```

Comment:
Leaving this second counter code for now because I may want to display some debug information every second, I could maybe move the fps counter output to every second rather than every frame